kotlin-browser/src/main/kotlin/org/w3c/dom/HTML.kt

```diff
@@ -0,0 +1,351 @@
+
+    inline val small: TagName<HTMLElement>
+        get() = "small".unsafeCast<TagName<HTMLElement>>()
+
+    inline val source: TagName<HTMLSourceElement>
+        get() = "source".unsafeCast<TagName<HTMLSourceElement>>()
+
+    inline val span: TagName<HTMLSpanElement>
+        get() = "span".unsafeCast<TagName<HTMLSpanElement>>()
+
+    inline val strong: TagName<HTMLElement>
+        get() = "strong".unsafeCast<TagName<HTMLElement>>()
+
+    inline val style: TagName<HTMLStyleElement>
+        get() = "style".unsafeCast<TagName<HTMLStyleElement>>()
+
+    inline val sub: TagName<HTMLElement>
+        get() = "sub".unsafeCast<TagName<HTMLElement>>()
+
+    inline val summary: TagName<HTMLElement>
+        get() = "summary".unsafeCast<TagName<HTMLElement>>()
+
+    inline val sup: TagName<HTMLElement>
+        get() = "sup".unsafeCast<TagName<HTMLElement>>()
+
+    inline val table: TagName<HTMLTableElement>
+        get() = "table".unsafeCast<TagName<HTMLTableElement>>()
+
+    inline val template: TagName<HTMLTemplateElement>
+        get() = "template".unsafeCast<TagName<HTMLTemplateElement>>()
+
+    inline val tbody: TagName<HTMLTableSectionElement>
+        get() = "tbody".unsafeCast<TagName<HTMLTableSectionElement>>()
+
+    inline val td: TagName<HTMLTableCellElement>
+        get() = "td".unsafeCast<TagName<HTMLTableCellElement>>()
+
+    inline val textarea: TagName<HTMLTextAreaElement>
+        get() = "textarea".unsafeCast<TagName<HTMLTextAreaElement>>()
+
+    inline val tfoot: TagName<HTMLTableSectionElement>
+        get() = "tfoot".unsafeCast<TagName<HTMLTableSectionElement>>()
+
+    inline val th: TagName<HTMLTableCellElement>
+        get() = "th".unsafeCast<TagName<HTMLTableCellElement>>()
+
+    inline val thead: TagName<HTMLTableSectionElement>
+        get() = "thead".unsafeCast<TagName<HTMLTableSectionElement>>()
+
+    inline val time: TagName<HTMLTimeElement>
+        get() = "time".unsafeCast<TagName<HTMLTimeElement>>()
+
+    inline val title: TagName<HTMLTitleElement>
+        get() = "title".unsafeCast<TagName<HTMLTitleElement>>()
+
+    inline val tr: TagName<HTMLTableRowElement>
+        get() = "tr".unsafeCast<TagName<HTMLTableRowElement>>()
+
+    inline val track: TagName<HTMLTrackElement>
+        get() = "track".unsafeCast<TagName<HTMLTrackElement>>()
+
+    inline val u: TagName<HTMLElement>
+        get() = "u".unsafeCast<TagName<HTMLElement>>()
+
+    inline val ul: TagName<HTMLUListElement>
+        get() = "ul".unsafeCast<TagName<HTMLUListElement>>()
+
+    inline val `var`: TagName<HTMLElement>
+        get() = "var".unsafeCast<TagName<HTMLElement>>()
+
+    inline val video: TagName<HTMLVideoElement>
+        get() = "video".unsafeCast<TagName<HTMLVideoElement>>()
+
+    inline val wbr: TagName<HTMLElement>
+        get() = "wbr".unsafeCast<TagName<HTMLElement>>()
+
+    inline val webview: TagName<HTMLElement>
+        get() = "webview".unsafeCast<TagName<HTMLElement>>()
```

Comment:
Why we use `unsafeCast` if we have inline factory method for creation of such tag names?

Reply:
Fixed in #1558